src/main/app/src/assets/i18n/nl.json

```diff
@@ -420,6 +420,12 @@
   "indicatie.VERGRENDELD": "is vergrendeld",
   "indicatie.VERLENGD": "is verlengd",
   "indicatie.VERZONDEN": "is verzonden",
+  "indicatie.IN_ONDERZOEK": "in onderzoek",
```

Comment:
Should being more usage specific here be helpful? adding something with 'persoon' in the label?